spec/unpacker_spec.rb

```diff
@@ -1,6 +1,257 @@
+# encoding: ascii-8bit
+
+require 'stringio'
+require 'tempfile'
+
 require 'spec_helper'
 
 describe MessagePack::Unpacker do
+  let :unpacker do
+    MessagePack::Unpacker.new
+  end
+
+  let :packer do
+    MessagePack::Packer.new
+  end
+
+  # TODO initialize
```

Comment:
Do what the TODO says or remove the comment. Don't merge with TODOs in the code.